### PR TITLE
Core: NEW station map

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Nanotrasen has introduced a new station into the ranks! Core is a 30-60 medium population map featuring a singularity in the center along with an off-branching arrivals area and two solar panel arrays.
**Station-exclusive jobs include:**
- Boxer
- Zookeeper
- Reporter

The evacuation shuttle used here is Rod, also designed and added into the game by me. The PR is #17129

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Map updated on 16/07/2023
Core-0

**Changelog**

:cl: Ubaser
- add: Nanotrasen has hired a group of architects to change up their selection with the introduction of the new mid-pop space station, Core!

### CODE REVIEW

**Comment by forgotmyotheraccount** [comment]
What happens when Arrivals runs out of EVA suits?

**Comment by UbaserB** [comment]
> What happens when Arrivals runs out of EVA suits?

i might add a small disposals section for people so they can return the suits to arrivals. Otherwise, just run through space you'll live :)

**Comment by lzk228** [comment]
goofy ahh arrivals

**Comment by deltanedas** [comment]
the singulo window grilles arent shocked so singuloose will be ridiculously easy

**Comment by Cheackraze** [comment]
A few things from a very brief look over:

That arrivals is unique but isn't going to work.  New players especially will be confused and punished by the forced spacewalk, etc.
The solar array is massive and boring, should split it into multiples and make them more interesting
Serious lack of maints: med and sci especially have none, and the maints that do exist are straight, narrow, and empty
Station specific jobs: these are *station specific*, meaning intended for their specific stations.

Looks like you've put in a good effort for it though so I can do a more in depth review when it's ready

**Comment by EmoGarbage404** [comment]
this map has next to 0 maintenance of any kind, which is a major problem.

also, check is right about arrivals. neat concept, but you're really throwing new players to the wolves. Many people aren't going to be able to go spacewalk to the main station a few minutes after joining.

**Comment by Cheackraze** [comment]
Also if you're including a custom evac shuttle, you should include the render of that as well since it will also need to be reviewed

**Comment by Emisse** [comment]
reviewed minor things on discord, will do a more in depth design review later today or tomorrow after work
check cartography threads

**Comment by potato1234x** [comment]
no robotics?

**Comment by Yousifb26** [comment]
Along with non electrified grills it doesn’t have plasma glass or shutters so you WILL die if you stay too long in maints with a singulose active. I suggest plasma glass to add a cool effect of being able to see a singulose from maints

**Comment by UbaserB** [comment]
> Along with non electrified grills it doesn’t have plasma glass or shutters so you WILL die if you stay too long in maints with a singulose active. I suggest plasma glass to keep the cool effect of being able to see a singulose from maints

this is actually on purpose, i added a set of glass, plasma sheets and rods so that engineering can do it themselves :)

**Comment by UbaserB** [comment]
> Also if you're including a custom evac shuttle, you should include the render of that as well since it will also need to be reviewed

i have a separate PR that should be accepted soon

**Comment by Cheackraze** [comment]
> > Along with non electrified grills it doesn’t have plasma glass or shutters so you WILL die if you stay too long in maints with a singulose active. I suggest plasma glass to keep the cool effect of being able to see a singulose from maints
> 
> this is actually on purpose, i added a set of glass, plasma sheets and rods so that engineering can do it themselves :)

Radiation in its current implementation is really overly punishing, particularly since it is functionally undetectable unless you are an engineer.  It is incredibly frustrating for players who die to something they are not aware is even happening or not aware of what to do etc.  If radiation was better conveyed to players through like green glowy stuff or in some other way this wouldn't be an issue, but as it stands, you're just going to end up killing 1/4 of the crew every shift and the victims will probably not be very happy about it

**Comment by UbaserB** [comment]
> > > Along with non electrified grills it doesn’t have plasma glass or shutters so you WILL die if you stay too long in maints with a singulose active. I suggest plasma glass to keep the cool effect of being able to see a singulose from maints
> > 
> > 
> > this is actually on purpose, i added a set of glass, plasma sheets and rods so that engineering can do it themselves :)
> 
> Radiation in its current implementation is really overly punishing, particularly since it is functionally undetectable unless you are an engineer. It is incredibly frustrating for players who die to something they are not aware is even happening or not aware of what to do etc. If radiation was better conveyed to players through like green glowy stuff or in some other way this wouldn't be an issue, but as it stands, you're just going to end up killing 1/4 of the crew every shift and the victims will probably not be very happy about it

Hm yeah that makes a lot of sense… i might add signs or something similar to warn anyone passing through

**Comment by UbaserB** [comment]
Im having a stroke and the ghost conflict is not going away

**Comment by UbaserB** [comment]
im confused i think i did something wrong

**Comment by UbaserB** [comment]
i was simply trying to update my map guh

**Comment by UbaserB** [comment]
i was simply trying to update my map, but im going to just reopen it real quickly

**Comment by UbaserB** [comment]
i was simply trying to update the thing, still new to github. Reopening now

File: Resources/Prototypes/Maps/core.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            HeadOfPersonnel: [ 1, 1 ]
+            ChiefEngineer: [ 1, 1 ]
+            StationEngineer: [ 3, 5 ]
+            ChiefMedicalOfficer: [ 1, 1 ]
+            MedicalDoctor: [ 3, 4 ]
+            Chemist: [ 2, 3 ]
+            ResearchDirector: [ 1, 1 ]
+            Scientist: [ 3, 4 ]
+            HeadOfSecurity: [ 1, 1 ]
+            SecurityOfficer: [ 4, 6 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:40]:
```suggestion
            SecurityOfficer: [ 4, 4 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            Bartender: [ 2, 2 ]
+            Botanist: [ 3, 3]
+            Chef: [ 1, 2 ]
+            Clown: [ 1, 1 ]
+            Janitor: [ 1, 2 ]
+            Mime: [ 1, 2 ]
+            Captain: [ 1, 1 ]
+            HeadOfPersonnel: [ 1, 1 ]
+            ChiefEngineer: [ 1, 1 ]
+            StationEngineer: [ 3, 5 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:33]:
```suggestion
            StationEngineer: [ 4, 4 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            - Passenger
+          availableJobs:
+            CargoTechnician: [ 2, 4 ]
+            Passenger: [ -1, -1 ]
+            Bartender: [ 2, 2 ]
+            Botanist: [ 3, 3]
+            Chef: [ 1, 2 ]
+            Clown: [ 1, 1 ]
+            Janitor: [ 1, 2 ]
+            Mime: [ 1, 2 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:29]:
```suggestion
            Mime: [ 1, 1 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          overflowJobs:
+            - Passenger
+          availableJobs:
+            CargoTechnician: [ 2, 4 ]
+            Passenger: [ -1, -1 ]
+            Bartender: [ 2, 2 ]
+            Botanist: [ 3, 3]
+            Chef: [ 1, 2 ]
+            Clown: [ 1, 1 ]
+            Janitor: [ 1, 2 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:28]:
```suggestion
            Janitor: [ 2, 2 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/emergency_rod.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+            CargoTechnician: [ 2, 4 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:22]:
```suggestion
            CargoTechnician: [ 3, 3 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
+- type: gameMap
+  id: Core
+  mapName: 'Core'
+  mapPath: /Maps/core.yml
+  minPlayers: 40
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:5]:
```suggestion
  minPlayers: 30
```

File: Resources/Prototypes/Maps/core.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  maxPlayers: 70
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:6]:
```suggestion
  maxPlayers: 60
```

File: Resources/Prototypes/Maps/core.yml [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+            HeadOfSecurity: [ 1, 1 ]
+            SecurityOfficer: [ 4, 6 ]
+            Chaplain: [ 1, 2 ]
+            Warden: [ 1, 1 ]
+            Librarian: [ 1, 1 ]
+            Lawyer: [ 2, 2 ]
+            Quartermaster: [ 1, 1 ]
+            SalvageSpecialist: [ 3, 4 ]
+            Musician: [ 1, 1 ]
+            AtmosphericTechnician: [ 3, 3 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:48]:
```suggestion
            AtmosphericTechnician: [ 2, 2 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+            ResearchDirector: [ 1, 1 ]
+            Scientist: [ 3, 4 ]
+            HeadOfSecurity: [ 1, 1 ]
+            SecurityOfficer: [ 4, 6 ]
+            Chaplain: [ 1, 2 ]
+            Warden: [ 1, 1 ]
+            Librarian: [ 1, 1 ]
+            Lawyer: [ 2, 2 ]
+            Quartermaster: [ 1, 1 ]
+            SalvageSpecialist: [ 3, 4 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:46]:
```suggestion
            SalvageSpecialist: [ 3, 3 ]
```

File: Resources/Prototypes/Maps/core.yml [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+            ChiefEngineer: [ 1, 1 ]
+            StationEngineer: [ 3, 5 ]
+            ChiefMedicalOfficer: [ 1, 1 ]
+            MedicalDoctor: [ 3, 4 ]
+            Chemist: [ 2, 3 ]
+            ResearchDirector: [ 1, 1 ]
+            Scientist: [ 3, 4 ]
+            HeadOfSecurity: [ 1, 1 ]
+            SecurityOfficer: [ 4, 6 ]
+            Chaplain: [ 1, 2 ]
```
Comment (Emisse) [inline Resources/Prototypes/Maps/core.yml:41]:
```suggestion
            Chaplain: [ 1, 1 ]
```

**Review by Emisse — CHANGES_REQUESTED** [review]

**Review by Emisse — APPROVED** [review]

**Pull request merged**